### DISCUSSION TITLE
renovate-config-validator passed even if renovate configuration is invalid JSON5

### DISCUSSION THREAD

**Discussion by suzuki-shunsuke** [renovatebot/renovate#13360] [Request Help] [answered]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Please select which platform you are using if self-hosting.

github.com

### Describe the bug

`renovate-config-validator` passed even if renovate.json5 is invalid JSON5.

Please see the repository. https://github.com/suzuki-shunsuke/test-renovate-config-validator

### Relevant debug logs

It seems that `renovate-config-validator` can't find `renovate.json5`.

<details><summary>Logs</summary>

```
$ LOG_LEVEL=debug ./node_modules/.bin/renovate-config-validator
DEBUG: Using RE2 as regex engine
DEBUG: Checking for config file in /Users/shunsuke-suzuki/repos/src/github.com/suzuki-shunsuke/test-renovate-config-validator/config.js
DEBUG: No config file found on disk - skipping
 INFO: Config validated successfully
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by suzuki-shunsuke** [comment]
BTW, I can't find the document about `renovate-config-validator`, and `renovate-config-validator` doesn't support help command and help option, so I don't know the specification of `renovate-config-validator`.

```console
$ ./node_modules/.bin/renovate-config-validator --help                       
 INFO: Config validated successfully

$ ./node_modules/.bin/renovate-config-validator help  
 INFO: Config validated successfully
```

**Comment by viceice** [comment, marked as answer]
You need to point `RENOVATE_CONFIG_FILE` env to your  file to check. See https://github.com/renovatebot/.github/blob/03cb56ff31a80b78b224e6024508007f502d77b6/.github/workflows/test.yml#L21 for sample workflow.

Also https://github.com/rinchsan/renovate-config-validator action isn't maintained by us, so ask there for help. We don't know what is happening there

**Reply by suzuki-shunsuke** [reply]
Thank you. I got it.
